File: Getting-Practice-with-Python-and-Javascript/4- Functions/Python/Excercises/program-5.py
# 5.  Escribe una función que tome una lista de números y devuelva la suma de todos los números en la lista.


lista_de_numero = [1997,12,3]


def suma_de_lista(lista):
    resultado = 0
    for c in lista:
        resultado += c;
    return f'El resultado de la suma de la lista {lista} es: {resultado}';

print(suma_de_lista(lista_de_numero))

File: Getting-Practice-with-Python-and-Javascript/4- Functions/Javascript/Excercises/program-5.js
// 5.  Escribe una función que tome una lista de números y devuelva la suma de todos los números en la lista.


let listaDeNumeros = [1997, 12, 3]



function sumaDeLista(lista) {
    let start = 0;
    let end = lista.length
    let resultado = 0;
    for (let i = start; i < end; i++) {
        resultado += lista[i];
    }
    console.log("Todo perfecto!")
    return resultado;
}

console.log(sumaDeLista(listaDeNumeros))